worker-batch-ingestion-plugin/pom.xml

```diff
@@ -55,6 +55,11 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.cafapi</groupId>
+            <artifactId>caf-logging</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
This should be added to the worker's container rather then to the plugin.

Reply:
Moved